rclpy/rclpy/node.py

```diff
@@ -1949,3 +1950,22 @@ def get_subscriptions_info_by_topic(
             topic_name,
             no_mangle,
             _rclpy.rclpy_get_subscriptions_info_by_topic)
+
+    def wait_for_node(
+        self,
```

Comment:
Another approach we could take is to put the implementation into a free function, living in it's own module (e.g. `graph.py`), that takes a node as an argument. This makes it easier to do dependency injection on node-like objects (nice for testing). We can can still have node method too, that just calls the free function.

I think this is a very minor comment that can be ignored; it's easy enough to break out the implementation later without breaking API.